android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt

```diff
@@ -38,7 +39,8 @@ data class QuestionnaireConfig(
   val groupResource: GroupResourceConfig? = null,
   val taskId: String? = null,
   val saveDraft: Boolean = false,
-  val snackBarMessage: SnackBarMessageConfig? = null
+  val snackBarMessage: SnackBarMessageConfig? = null,
+  val carePlan: List<CarePlanConfig> = emptyList()
```

Comment:
think we should change name
```suggestion
  val carePlanConfigs: List<CarePlanConfig> = emptyList()
```